index.js

```diff
@@ -24,6 +24,12 @@ program
       }
     }
 
+    var num = numberOfResults;
```

Comment:
Can we just use one variable instead of create a new one for number of results?

Also semantically, can we list this as resultsToDisplay or something? numberOfResults sounds like number of results returned to me.

Reply:
we should also avoid the use of `var` and use `let` only if needed, more on that below